torchtune/training/_distributed.py

```diff
@@ -280,7 +280,7 @@ def lora_wrap_fsdp(module: nn.Module, recurse: bool, **kwargs):
 
 
 def load_from_full_model_state_dict(
-    model: "FSDPModule",
+    model: "FSDPModule",  # noqa
```

Comment:
this seems a bit off, since we only import FSDPModule a few lines later. So it means that it may be something that is not an FSDPModule. Is that right?

Not related to this PR, but maybe the qa shouldnt be supressed.

Reply:
Probably because the import line is not at the top so the linter doesn't like this. I'll leave the noqa for now and we can address in a follow up